cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerMgmtTelemetryService.java

```diff
@@ -82,15 +82,15 @@ public void setupTelemetryRole(final Stack stack, final ApiClient client, final
         if (isWorkflowAnalyticsEnabled(stack, telemetry)) {
             WorkloadAnalytics workloadAnalytics = telemetry.getWorkloadAnalytics();
             ClouderaManagerResourceApi cmResourceApi = new ClouderaManagerResourceApi(client);
-            ApiConfigList apiConfigList = buildTelemetryCMConfigList(workloadAnalytics);
+            final String credentialName = String.format(ALTUS_CREDENTIAL_NAME_PATTERN, UUID.randomUUID().toString());
```

Comment:
If the user is in CM, then would be it is easier to check that if the user exists then do not create it again?

(If the user would have been added to Altus, then shouldn't we store the generated user in the database to make sure that if the flow is restarted then we create the same user and if it needs to be deleted then we know what needs to be deleted)

Reply:
it's only CM cb, im checking only updateAccount call maybe that does not require user should exist

Reply:
unfortunately update throws exception if account does not exist, so i need to query accounts first, PR updated for this